frontend/src/components/market-time.tsx

```diff
@@ -1,33 +1,36 @@
-import { cn } from "@/lib/utils";
+import { cn } from "../../lib/utils";
 
 interface MarketTimeProps {
-    endTime: bigint;
-    className?: string;
+  endTime: bigint;
+  className?: string;
 }
 
 const formatDate = (dateString: string) => {
-    return new Date(dateString).toLocaleDateString('en-US', {
```

Comment:
please remove your IDE eslint settings , this makes it hard to review the real changes